openapi/openapi.yaml

```diff
@@ -931,6 +931,28 @@ paths:
         "404":
           description: The given build log does not exist.
 
+  /kebechet:
+    post:
+      tags: [Kebechet]
+      x-openapi-router-controller: thoth.user_api.api_v1
+      operationId: schedule_kebechet
+      summary: Schedule kebechet instance from webhook
+      requestBody:
+        required: true
+        description: Body of a git service webhook
+        content:
+          application/json:
+            schema:
+              type: object
+              additionalProperties: true
+      responses:
+        "200":
```

Comment:
Please respond with 202 (Accepted) HTTP status code as the analysis was just accepted.